src/olympia/reviewers/views.py

```diff
@@ -1201,8 +1202,9 @@ def download_git_stored_file(request, version_id, filename):
         raise http.Http404()
 
     actual_blob = serializer.git_repo[blob_or_tree.oid]
+
     response = http.FileResponse(
-        memoryview(actual_blob),
+        io.BytesIO(memoryview(actual_blob)),
```

Comment:
That shouldn't be in here.

Reply:
rebased and force pushed to get rid of the extra commit